src/components/hero/_macro.spec.js

```diff
@@ -105,13 +105,32 @@ describe('macro: hero', () => {
         expect(content).toBe('Topic Text');
     });
 
-    it('outputs the official statistics badge when officialStatisticsBadge is set to true', () => {
-        const $ = cheerio.load(renderComponent('hero', { ...EXAMPLE_HERO, officialStatisticsBadge: true }));
+    it('outputs the official statistics badge when officialStatisticsBadge is set to true for `grey` variant', () => {
```

Comment:
```suggestion
    it('outputs the official statistics badge when officialStatisticsBadge is set to true and variants is set to "grey" ', () => {
```
I think this wording describes the situation a bit better